client/deposits/details/style.scss

```diff
@@ -8,17 +8,19 @@
 		height: 0.85em;
 
 		&.is-paid {
-			background: $studio-green-50;
-			border-color: $studio-green-5;
+			background: $wp-green-70;
+			border-color: $wp-green-0;
 		}
+
 		&.is-pending,
```

Comment:
>I couldn't find a canceled state showing in grey in the issue description.

There is an image shared in the issue with slightly different brighter shades ( probably studio ) . I had to recopy this image here since it had some weird private URL. 

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/c102c905-cc7b-4478-8004-25977e15b5c1)

We may want to use the [wp-admin shade equivalents](https://github.com/Automattic/woocommerce-payments/blob/9d8c275a4cca821f06595e71918b531ed5b5897c/client/stylesheets/abstracts/_colors.scss#L17), that are slightly darker / a bit dull compared to the screenshot in the issue. @rogermattic may kindly reconfirm. 

> Secondly, in transit and pending states are both yellow.

As per the `Acceptable` criteria in the issue, we need to use shades of blue for in transit. 

So maybe 

```suggestion
		&.is-pending {
		    background: $wp-blue-70;
		    border-color: $wp-blue-0;
		}
		
		&.is-canceled {
		    background: $wp-gray-70;
		    border-color: $wp-gray-blue;
		}
```
The gray color may end up looking almost black, though.

![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/704430e1-877b-458a-8846-a33bc92e62e2)

We probably need a different shade?